SparkPost/2web2ui
# AC-629: Hides SSO checkbox if SSO is not configured

### What Changed
 - Hid SSO checkbox on user Edit form if SSO is not configured

### How To Test
 - Check account with SSO enabled to see if checkbox is visible and enabled (appteam works)
 - Check account without SSO configured to see if checkbox is hidden

### To Do
- []
